Source/Classes/VSTeamVersions.ps1

```diff
@@ -8,6 +8,7 @@ class VSTeamVersions {
    static [string] $Git = '3.0'
    static [string] $DistributedTask = '3.0-preview'
    static [string] $VariableGroups = '3.2-preview.1'
+   static [string] $TaskGroups = '5.1-preview.1'
```

Comment:
This should be the version number supported by TFS 2017. If it is not supported in TFS 2017 it should default to "". When the functions are called on an unsupported version the functions must error letting the caller know they are not supported.

Reply:
Not sure what value we want here.  I see that [2017 supports 3.x](https://docs.microsoft.com/en-us/azure/devops/integrate/concepts/rest-api-versioning?view=azure-devops), so I think we want the latest version of 3.x, right?  Not sure how to determine the value of x.  When I nav to the [taskgroups API page](https://docs.microsoft.com/en-us/rest/api/azure/devops/distributedtask/taskgroups?view=azure-devops-rest-5.1), I was expecting to see all versions, but it stops at 5.1.

Reply:
I will see what I can find out. If I determine it is not supported in TFS 2017 we will leave "" and throw when someone tries to use in an unsupported version.

Reply:
I found all the versions for each server and updated the APIs